modules/charon-core/src/main/java/org/wso2/charon3/core/objects/Group.java

```diff
@@ -83,8 +84,11 @@ public void setDisplayName(String displayName) throws CharonException, BadReques
             for (Attribute subValue : subValuesList) {
                 ComplexAttribute complexAttribute = (ComplexAttribute) subValue;
                 Map<String, Attribute> subAttributesList = complexAttribute.getSubAttributesList();
-                memberList.add(((SimpleAttribute) (subAttributesList.get(
-                                SCIMConstants.CommonSchemaConstants.VALUE))).getValue());
+                if(subAttributesList != null && subAttributesList.containsKey(
```

Comment:
Formatting issue

Reply:
@IsuraD  can you bit explain about the formatting issue

Reply:
There should be a space after the "if condition" as follows. 

if (subAttributesList != null && subAttributesList.containsKey(